packages/xod-cloud-compile/src/compile.js

```diff
@@ -46,9 +46,16 @@ const compile = R.curry((hostname, accessToken, suite, opts) => {
     )
     .then(res => {
       if (!res.ok) {
-        return res.json().then(json => Promise.reject(
-          createError(EC.WASM_COMPILATION_ERROR, json)
-        ));
+        return res.json()
```

Comment:
LGTM, except uncatched JSON parsing error.

Reply:
Nice catch! Just fixed it.